internal/util/terraform.go

```diff
@@ -189,6 +212,7 @@ func (t *TerraformBinUtil) Apply(ctx context.Context, tfVars []string) error {
 	}
 
 	argv := []string{"apply", "-no-color"}
+	argv = append(argv, args...)
```

Comment:
re: `cfg.BatchMode` -- is this redundant now, that we have the the `--tf-apply-options`? What happens if someone sets `--batch-mode --tf-apply-options "-auto-approve=false"`?

As a side note, if we keep `BatchMode`, it should also set `-input=false`, which prevents TF from prompting you for missing tfvar definitions.
https://www.terraform.io/docs/commands/apply.html#input-true

Reply:
Good point on the input. I'm really hoping that the options would be scoped to specify a backend. Again, if you're a wicked advanced terraform user and really know what you're doing, use terraform module and not the DCE CLI.